imagepicker/src/main/java/com/esafirm/imagepicker/features/ImagePickerConfig.kt

```diff
@@ -30,6 +31,7 @@ class ImagePickerConfig(
     var showDoneButtonAlways: Boolean = false
 ) : BaseConfig(), Parcelable {
 
+    @IgnoredOnParcel
```

Comment:
Question: is `@Transient` still included in parcelize's pacel?

Reply:
Im not sure about `@Transient`, based on this [SO answer](https://stackoverflow.com/a/54434904), it has no effect on parcelable project. What im sure is property that is not in primary constructor will be ignored since it is mentioned in [official docs](https://developer.android.com/kotlin/parcelize) of kottlin parcelize.